utils/README.md

```diff
@@ -99,3 +100,40 @@ notebooks$ bash utils/dask-setup.sh 8 8786 8787 8790 12.34.567.890 MASTER DEBUG
 ```
 
 Note: `LOG` is an optional argument.
+
+## split-data-mortgage
+
+`split-data-mortgage.sh` is designed to accept a single argument: `SIZE`. `SIZE` is an integral value specifying the target partition file size. Because this script uses `split`, `SIZE` may also have units (K, M, G, T, P, E, Z, Y ... powers of 1024; KB, MB, ... powers of 1000) (e.g.) `10K == 10 * 1024`
+
+You must update the paths to data near the beginning of the script:
+
+```bash
+#!/bin/bash
+set -e
+
+SRC="/path/to/mortgage/perf"
+DST="/path/to/mortgage/perf_split"
+WORK="/path/to/mortgage/"
+...
+```
+
+* `SRC`: path to the original mortgage data files
+* `DST`: path to save the split files
+* `WORK`: working directory (where both data directories will be visible)
+
+Note: this script corrects for record spilling (when a record data appears in two files). Here's some sample output:
+
+```bash 
+[Mon Dec 17 23:14:44 UTC 2018] Processing Performance_2003Q3.txt_1_0_0
+[Mon Dec 17 23:14:44 UTC 2018]   Sorting...
+[Mon Dec 17 23:15:03 UTC 2018]   Splitting...
+[Mon Dec 17 23:15:22 UTC 2018]   Split into 2 files
+[Mon Dec 17 23:15:30 UTC 2018]   Checking for spill...
+[Mon Dec 17 23:15:30 UTC 2018]     >>Spill detected, correcting<<
+```
+
+The script follows this procedure:
+1. Sort the data in the file
+2. Split the file into `N` parts which are not larger than `SIZE`
```

Comment:
This is a nitpick, the files can be larger than SIZE if there is a spill. So its more "Split the file in N parts which are approximately SIZE".